Metadata.yaml

```diff
@@ -0,0 +1,278 @@
+        will depend on how active the supervision is. If most of the supervision
+        is coming from the Core developers then Contributor status is
+        appropriate. This will be discussed with the person supervising the Core
+        developer and/or Developers.
+
+      This author list is ordered by last name.
+    List:
+      - Name: Armaza, Cristóbal
+        Orcid: 0000-0002-1791-0743
+        Affiliations: [*cornell]
+        GitHub: carmaza
+      - Name: Bonilla, Gabriel S.
+        Orcid: 0000-0003-4502-528X
+        Affiliations: [*cornell]
+        GitHub: gsb76
+      - Name: Hébert, François
+        Orcid: 0000-0001-9009-6955
+        Affiliations: [*caltech]
+        GitHub: fmahebert
+      - Name: Kumar, Prayush
+        Orcid: 0000-0001-5523-4603
+        Affiliations: [*cornell]
+        GitHub: prayush
+      - Name: Lovelace, Geoffrey
+        Orcid: 0000-0002-7084-1070
+        Affiliations: [*fullerton]
+        GitHub: geoffrey4444
+      - Name: Moxon, Jordan
+        Orcid: 0000-0001-9891-8677
+        Affiliations: [*caltech]
+        GitHub: moxcodes
+      - Name: O'Shea, Eamonn
+        Affiliations: [*cornell]
+        GitHub: osheamonn
+      - Name: Pfeiffer, Harald P.
+        Orcid: 0000-0001-9288-519X
+        Affiliations: [*aei]
+        GitHub: haraldp271
+      - Name: Scheel, Mark A.
+        Orcid: 0000-0001-6656-9134
+        Affiliations: [*caltech]
+        GitHub: markscheel
+      - Name: Teukolsky, Saul A.
+        Orcid: 0000-0001-9765-4526
+        Affiliations: [*caltech, *cornell]
+        GitHub: teukolsky
+
+  Contributors:
+    Description: |
+      Anyone who has contributed to SpECTRE will be added to this list and
+      remain a Contributor indefinitely unless they wish to be removed.
+
+      Guidelines on becoming a Contributor:
+
+      - See our guidelines in `docs/CONTRIBUTING.md` for information on how to
+        get involved with SpECTRE development.
+      - Having any PR merged qualifies for this list. If multiple people worked
+        on the PR together, e.g. several undergraduate students contributing to
+        a single PR, all persons involved will become a Contributor.
+      - Supervising a project that results in a merged PR qualifies for this
+        list. The supervision must be active, i.e. not primarily done by the
+        Core developers.
+      - Having an idea for a contribution does not count towards any contributor
+        status. There must be active supervision of the person(s) implementing
+        the idea, such as advice on debugging, how to best implement the idea,
+        etc. The supervision is what will be counted for contributor status.
+        Generally, a paper is written where the person who came up with the idea
+        is a co-author, which is how credit is given for the idea. An example
+        would be suggesting "We should try reconstruction scheme X" without
```

Comment:
maybe `would be` -> `is`? I found the occurrence of `would` twice in this sentence makes it a bit more difficult to read than necessary. (My fault, I think I wrote this)